repos/system_upgrade/el8toel9/actors/nisscanner/libraries/nisscan.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from leapp.libraries.stdlib import api
+from leapp.models import NISConfig
+
+PACKAGES_NAMES = ('ypserv', 'ypbind')
+YPBIND_CONF_FILE = '/etc/yp.conf'
+YPSERV_DIR_PATH = '/var/yp'
+YPSERV_DEFAULT_FILES = ('binding', 'Makefile', 'nicknames')
+
+
+class NISScanLibrary(object):
```

Comment:
Nit, starting with Python 3 one writes classes again like this:
```suggestion
class NISScanLibrary:
```

Reply:
Also looking at this class, I don't see why this is a class at all. It doesn't use any shared variables. These could be just 3 free standing functions.

Reply:
This what was recommend for me to do. Comment from March 12:

> in the check phase, actors must not communicate with the system. split it into 2 actors (aka scanner, checker) where the scanner gets all required data, propagates them via a msg, and the check actor does additoinal actions.

Reply:
I think what @vinzenz means is stylistic "there is no need for a class here, just use functions".

Reply:
@vinzenz @mkulik-rh I suggest leave it be (leave the object). Our linter needs to be taught to treat different actors (like py3 only for 8to9) differently first, now it applies generic rules to 7to8 and 8to9. 
It is a NIT anyway in my opinion.

Reply:
It is, that's the first word in the original comment :-)

Reply:
Changed.